enterprise/tasks.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Django tasks.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from logging import getLogger
+
+from celery import shared_task
+
+from enterprise.models import EnterpriseCourseEnrollment
+
+LOGGER = getLogger(__name__)
+
+
+@shared_task
+def create_enterprise_enrollment(course_id, enterprise_customer_user):
+    """
+    Create enterprise enrollment for user if course_id part of catalog for the ENT customer.
+    """
+    enterprise_customer = enterprise_customer_user.enterprise_customer
+    if enterprise_customer.catalog_contains_course(course_id):
+        LOGGER.info((
+            "Creating EnterpriseCourseEnrollment for user %s "
+            "on course %s for enterprise_customer %s"
+        ), enterprise_customer_user.user_id, course_id, enterprise_customer)
+        EnterpriseCourseEnrollment.objects.create(
```

Comment:
maybe use a `get_or_create` here just to be safe? (so we don't create duplicates)

Reply:
clicking enroll in devstack did NOT create 2 enterprise course enrollments, so i do not think this is necessary

Reply:
Hmm, good question. Even though in testing with devstack, it did not create 2 enterprise course enrollments... I feel that it might be good to be safe here just in case with `get_or_create`. If there are any edge cases where an enrollment might already exist in stage/production data, might be good to proactively handle that.